packages/trace-viewer/src/ui/actionList.tsx

```diff
@@ -104,6 +104,29 @@ export const ActionList: React.FC<ActionListProps> = ({
   </div>;
 };
 
+const ActionContext: React.FC<{
```

Comment:
Context is heavily overloaded in our system. For example, `ActionInContext` is one of the main recorder types. I'd be in favor of calling things their names: orange "locator" and purple "parameters".